### PR TITLE
Release configs

### PR DESCRIPTION
**Pull request by ol-web**

### CODE REVIEW

File: vue.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/** @type {import("eslint").Linter.Config}  */
+const vue = {
+  extends: [
+    'plugin:vue/vue3-recommended',
+    'eslint:recommended',
+    '@sparing-software/eslint-config',
+    '@vue/eslint-config-typescript/recommended',
```
Comment (adrianklimek) [inline vue.js:9]: Isn't it conflicting with our eslint typescript configuration?

File: index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/** @type {import("eslint").Linter.Config}  */
+const defaultConfig = {
```
Comment (adrianklimek) [inline index.js:2]: Are all rules from the guide there?

**Review by adrianklimek — APPROVED** [review]

File: index.js (hunk 2 above)
Comment (ol-web) [reply index.js:2]: @adrianklimek i added ts guide rules for now, will do js guide next

File: vue.js (hunk 1 above)
Comment (ol-web) [reply vue.js:9]: @adrianklimek  switching order causes a lots of weird errors here so i extended the rules manually to take priority. alternatively i could remove the custom vue config and try to configure it from scratch but i'm not sure if that's a better idea.

**Pull request merged**